src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -295,4 +296,22 @@ public Action updateAction(final Action action) {
     }
     return existingAction;
   }
+
+  @Transactional(
+      propagation = Propagation.REQUIRED,
+      readOnly = false,
+      timeout = TRANSACTION_TIMEOUT)
+  public void rerunAction(final Action action) throws CTPException {
+    log.with("actionId", action.getId()).debug("Entering rerunAction");
```

Comment:
This probably should be put in snake_case so its consistent with the rest of the logging